docs/pages/2020_Intro_Week/scripts/pressure_convergence.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
```

Comment:
has this file been tested in some fashion, that it still works with the new aiida-core/aiida-quantum-espresso versions?